### ISSUE TITLE
Assign our service accounts to Google Compute Engine instances instead of activating them manually

### ISSUE THREAD

**Issue by cameel** [golemfactory/concent-deployment#372]
It's best to do this task after #369.

Currently we're managing service account keys along with other secrets. Turns out there is a way to move this responsibility to Google Cloud as long as the account is only used from a GCP instance. The docs say that `gcloud` can automatically use the service account assigned to an instance. This will make it harder to leak these keys.

See:
- [Creating and enabling service accounts for instances > Authenticating tools on an instance using a service account](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#tools)
- [Creating and enabling service accounts for instances > Using the Compute Engine Default Service Account](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#using_the_compute_engine_default_service_account)

1. Update `concent-builder` and `concent-deployment-server` metadata so that they use service accounts we've been activating there manually.
    - This requires a reboot.
    - You may need to also deactivate the accounts we've been using so far or reset `gcloud` settings.
2. Remove `gcloud auth activate-service-account` step from our playbooks.
    - From now on we'll be using the account assigned to an instance.
    - When running these playbooks on a machine outside of GCP, we'll now be using our personal accounts, activated manually.
        - This way the service account credentials will never have to get out of the machine and user accounts of all DevOps members of our team have enough privileges for such an operation and are already accessible on their machines.
        - Note that this is a rare case anyway. It would only be necessary when the **target** machine is outside of Google Cloud. When you're just starting the playbooks on your machine but they're executing stuff on `concent-builder`/`concent-deployment-server`, they're using the service accounts.
3. Remove service account keys from our secrets.
    - Also from secrets stored on any remote machines.
    - And ask other team members to update their secrets if they have them.
4. Update README.

**Comment by cameel** [comment]
@bartoszbetka Before you close this issue please confirm (in a comment) that you did all the non-implementation steps from the list above.

**Comment by bartoszbetka** [comment]
Steps that already done: `1, 2, 3 a), 4`.

**Comment by cameel** [comment]
It would be better if you just wrote what has not been done and why.
